go/api/base_client.go

```diff
@@ -744,3 +744,22 @@ func (client *baseClient) Del(keys []string) (Result[int64], error) {
 
 	return handleLongResponse(result)
 }
+
+func (client *baseClient) Copy(sourceKey, destinationKey string, destinationDB *int, replace bool) (Result[bool], error) {
```

Comment:
Behavior of `Copy` is different for standalone and cluster mode. Can we please keep the definition of commands consistent with other clients? [Reference](https://github.com/valkey-io/valkey-glide/blob/main/java/client/src/main/java/glide/api/BaseClient.java)

```
GenericBaseCommands:
Copy(source, destination)
Copy(source, destination, replace)

GenericCommands:
Copy(source, destination, destinationDB)
Copy(source, destination, destinationDB, replace)
```